verilog/formatting/formatter_test.cc

```diff
@@ -1574,6 +1574,20 @@ static constexpr FormatterTestCase kFormatterTestCases[] = {
      "      .in (ex_input_pins.in)\n"  // aligned
      "  );\n"
      "endmodule\n"},
+    {"module test;\n"
+     "bind entropy_src tlul_assert #(.EndpointType(\"Device\"))\n"
+     "tlul_assert_device (.clk_i, .rst_ni, .h2d(tl_i), .d2h(tl_o));\n"
+     "endmodule\n",
+     "module test;\n"
+     "  bind entropy_src tlul_assert #(\n"
+     "      .EndpointType(\"Device\")\n"
+     "  ) tlul_assert_device (\n"
+     "      .clk_i,\n"
+     "      .rst_ni,\n"
+     "      .h2d(tl_i),\n"
+     "      .d2h(tl_o)\n"
```

Comment:
Could we add some tests with:
* blank line in between
* comments in between

Reply:
+1